docs/pages/jpscore/jpscore_trains.md

```diff
@@ -5,189 +5,133 @@ tags: [jpscore, file, simulation]
-        <vertex px="10" py="0"/>
-      </polygon>
-      <polygon caption="wall" type="track-1">
-        <vertex px="0" py="3"/>
-        <vertex px="2" py="4"/>
-      </polygon>
-      <polygon caption="wall" type="track-1">
-        <vertex px="7" py="4"/>
-        <vertex px="8.00377" py="3.79962"/>
-      </polygon>
-      <polygon caption="wall" type="track-1">
-        <vertex px="8.00377" py="3.79962"/>
-        <vertex px="9.00063" py="3.53957"/>
-      </polygon>
+For using train in the simulation, tracks need to be defined in the geometry. 
+A track is a consecutive line sequence, marked by the type `type="track"`.
+For each track, one point needs to marked as starting point with `start="true"`. 
+
+### Example
+```xml
+<room id="1" caption="floor">
+    <subroom id="2" caption="Room 2" A_x="0" B_y="0" C_z="0">
+        ...
+        <polygon caption="wall" type="track" track_id="1">
+            <vertex px="-10" py="10" start="true" />
+            <vertex px="-5" py="10" />
+        </polygon>
+        <polygon caption="wall" type="track" track_id="1">
+            <vertex px="-5" py="10" />
+            <vertex px="0" py="10" />
+        </polygon>
+        ...
     </subroom>
-    <crossings/>
- </room>
+</room>
 ```
 
-## Train timetable
-
-### Definition 
-
-In this file the following information regarding a train are defined: 
-
-- Track where the train arrives, defines through two points: start and end of the track.
-- the start and end of the train. Trains are assumed to be linear, although tracks can have a curve.
-- Times of arrival and departure.
-
-{%include note.html content="train's coordinates are **relative** to the point `track_start`."%}
-
-### Sample
-
-```
- <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
- <train_time_table>
-  <train id="1" type="RE" room_id="0"
-      track_start_x="-10" track_start_y="10" 
-      track_end_x="0" track_end_y="10"
-      train_start_x="1" train_start_y="3"
-      train_end_x="7" train_end_y="3"
-      arrival_time="5" departure_time="15">
-  </train>
-  <train id="2" type="ICE" room_id="0"
-      track_start_x="0" track_start_y="10" track_end_x="10" track_end_y="10"
-      train_start_x="2" train_start_y="3"
-      train_end_x="12" train_end_y="3"
-      arrival_time="30" departure_time="50">
-  </train>
- </train_time_table>
-```
 ## Train types
-
 ### Definition
-
 A train is defined through the following information: 
 
-- type (string): for example RE or ICE.
+- type (string): Unique key to identify train
 - length (int): length of the train
-- capacity (int): maximal number of passengers
-- doors: a list of doors. Every door is defined by a 2D point.
-
-{%include note.html content="The parameter `length` is not used yet. Maybe it will be helpful for sanity checks. Actually, the length of the train is internally calculated as the difference between `train_start` and `train_end`."%}
-
-### Train example
+- agents_max (int): maximal number of passengers
```

Comment:
- `agents_max` (int): maximal number of passengers

Reply:
Done